### ISSUE TITLE
Aardvark.Base in WPF project not compiling on some machines

### ISSUE THREAD

**Issue by luithefirst** [vrvis/aardvark.base#26]
We have a WPF project with only an Aardvark.Base reference that installs DevilSharp (FSharp project)

During compile the project dependencies are tried to be resolved and complains about missing FSharp.Core 4.3.1.0 on some machines. NOTE: The Visual F# Tools 4.0 are installed.

"error MC1000: Unknown build error, 'Cannot resolve dependency to assembly 'FSharp.Core, Version=4.3.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' because it has not been preloaded. When using the ReflectionOnly APIs, dependent assemblies must be pre-loaded or loaded on demand through the ReflectionOnlyAssemblyResolve event"

[buildlog.txt](https://github.com/vrvis/aardvark.base/files/796087/buildlog.txt)

The Problem can be resolved by manually adding a FSharp.Core reference.

**Comment by haraldsteinlechner** [comment]
This is because devilsharp "uses" something of fsharp.core (don't know what, maybe fsharpfunction) but does not claim to use it. the usage is benign and i'm sure one could remove this depdendency by looking at the generated code. maybe aardvark.base could just reference FSharp.Core as package and the problem is gone. i have no idea consequences this would have and i think this is useless once dotnetcore is out. btw having a manual fsharp.core package reference in the wpf project works as well. 

in conclusion all measures to work around have potential problems i'm not aware of and the problem should be a no-problem when everything uses fsharp.core as package. i propose to simply add a fsharp.core reference when the wpf compiler asks for it and wait...

**Comment by krauthaufen** [comment]
resolved?

**Comment by haraldsteinlechner** [comment]
@luithefirst
